src/store/sparql.py

```diff
@@ -1,8 +1,12 @@
 
 from store.base import BaseStore
```

Comment:
so python imports are split into three blocks by convention.

```
<anything from the standard library>

<anything from a third party package>

<anything local>
```

And should be alphabetical within those blocks. So your imports should look like:

```
import os
from typing import Dict

from SPARQLWrapper import SPARQLWrapper, QueryResult

from store.base import BaseStore
```

Reply:
Im surprised this isn't something an IDE or linter would automatically correct, is this maybe something we need to look into as a team standard to make sure we each have configured? But by all means yes lets follow the conventions even if by hand for now